docs/src/outputformats.md

```diff
@@ -86,6 +86,28 @@ Literate can output markdown in different flavors. The flavor is specified using
    [CommonMark](https://commonmark.org/) specification.
  - `flavor = Literate.FranklinFlavor()`: this outputs markdown meant to be used as input
    to [Franklin.jl](https://franklinjl.org/).
+ - `flavor = Literate.QuartoFlavor()` : this outputs markdown file (with file extension ".qmd") meant to be used with [Quarto CLI](https://quarto.org).
+
+#### Quarto Flavor Tips & Tricks
+
+[Quarto](https://quarto.org/) is a new and exciting open-source scientific and technical publishing system, which can extend the range of output formats from your Literate.jl-formatted scripts.
+Literate.jl will produce a `.qmd` file, which can be used as input to Quarto CLI to produce a variety of output formats, including HTML, PDF, Word and RevealJS slides.
+
+Syntax tips:
+- `# `(hashtag followed by a space) at the beginning of a line will be stripped and anything following will rendered as a markdown, e.g., `# # Header level 1` in your script will be rendered as `# Header level 1` in your .qmd file (ie, it will show as a header). Use this for adding the YAML header at the top.
+- `#`(hashtag not followed by a space) at the beginning of a line will be stripped and anything following will be rendered as part of the code block. This is useful to provide Quarto commands in computation blocks, e.g., `## echo: false` in your script will be rendered as `#| echo: false` which will tell Quarto to not echo the code block in the output (see [Guide: Executions options](https://quarto.org/docs/computations/execution-options.html) for more commands).
```

Comment:
Where does the `|` come from?

Reply:
Good point! It dropped out from the snippet.

The goal was to keep consistency with the current syntax:
# xyz -> markdown
## code comment

So in the above example, it works like:
##! echo: false  -> #! echo: false (code block)

I've re-worded some of the examples and made this more explicit